include/eve/module/real/core/function/regular/simd/x86/convert_256.hpp

```diff
@@ -30,6 +30,14 @@ namespace eve::detail
     {
       return v0;
     }
+    else if constexpr( current_api >= avx2 && (N::value == 4)
+                       && std::is_integral_v<In> && (sizeof(In) == 8)
+                       && std::is_integral_v<Out> && (sizeof(Out) == 4) )
+    {
+      // u64 to u32
+      const auto p = _mm256_permutevar8x32_epi32(v0, _mm256_set_epi32(7, 5, 3, 1, 6, 4, 2, 0));
```

Comment:
`auto const` :p

Reply:
Fixed in 3ca5c9d3a2ad7b252bb269f11d6dbd14d3418520